src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31122,99 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22006647") obo:IAO_0000115 obo:CL_4033027 "An OFF diffuse bipolar cell that makes synaptic contact with both L/M and S-cone photoreceptors, and only minimal contact with rod photoreceptors.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033027 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033027 "2023-04-04T10:48:29Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033027 "DB1 cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22006647") Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033027 "A diffuse bipolar 1 cell on average makes 81 ribbon (output) synapses, 13 contacts are with OFF parasol cells and 47 contacts are with OFF midget cells. This cell might make flat contacts at both triad-associated and nontriad-associated positions on the cone pedicle.")
+AnnotationAssertion(rdfs:label obo:CL_4033027 "diffuse bipolar 1 cell")
+SubClassOf(obo:CL_4033027 obo:CL_0000750)
+
+# Class: obo:CL_4033028 (diffuse bipolar 2 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25309346") Annotation(oboInOwl:hasDbXref "PMID:27833534") obo:IAO_0000115 obo:CL_4033028 "An OFF diffuse bipolar cell that has preferential connectivity with ON parasol cells and bistratified amacrine cells. This cell contains a large number of synaptic ribbons and a small axon arbor area.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033028 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033028 "2023-04-04T10:49:07Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033028 "DB2 cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") rdfs:comment obo:CL_4033028 "A diffuse bipolar 2 cell makes an average of 133 ribbon synapses, 47 contacts are with OFF parasol cells and 4 contacts are with OFF midget cells.")
+AnnotationAssertion(rdfs:label obo:CL_4033028 "diffuse bipolar 2 cell")
+SubClassOf(obo:CL_4033028 obo:CL_0000750)
+
+# Class: obo:CL_4033029 (diffuse bipolar 3a cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasDbXref "PMID:32555229") obo:IAO_0000115 obo:CL_4033029 "An OFF calbindin-positive diffuse bipolar cell that has a large dendritic field and stratifies narrowly close to the middle of the Inner plexiform layer. Its axon terminal is characterized by regularly branching and varicose processes resembling beads on a string.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033029 <https://orcid.org/0000-0001-6677-8489>)
```

Comment:
Per PMID:32032773, the last sentence is applicable to DB3 cells (both a and b), so consider creating a new class 'diffuse bipolar 3 cell' or noting that this is not a uniquely distinguishing feature for this (BD3a) cell type. PMID:32032773 contains several distinctions between DB3a and DB3b which could be used.

Reply:
Thank you for the suggestion. I have added the new term and had db3a and db3b as child terms.